src/order/hom/complete_lattice.lean

```diff
@@ -3,7 +3,7 @@ Copyright (c) 2022 Yaël Dillies. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Yaël Dillies
 -/
-import order.complete_lattice
+import data.set.lattice
```

Comment:
This feels possibly wrong but I guess we always had an import in this direction so is out of scope

Reply:
Yep, we might want to turn this one around too at some point.